src/main/java/mekanism/client/gui/GuiLogisticalSorter.java

```diff
@@ -377,13 +382,16 @@ protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
         fontRenderer.drawString(LangUtils.localize("gui.filters") + ":", 11, 19, 0x00CD00);
         fontRenderer.drawString("T: " + tileEntity.filters.size(), 11, 28, 0x00CD00);
 
-        fontRenderer.drawString("RR:", 12, 74, 0x00CD00);
+        fontRenderer.drawString(LangUtils.localize("mekanism.gui.logisticalSorter.singleItem") + ":", 12, 48, 0x00CD00);
+        fontRenderer.drawString(LangUtils.localize("gui." + (tileEntity.singleItem ? "on" : "off")), 27, 60, 0x00CD00);
```

Comment:
Would you kindly change these on/off localisations to use `mekanism.common.util.LangUtils#transOnOff`?

Will save some trouble down the track when I fix the lang prefix issues